appium/webdriver/extensions/action_helpers.py

```diff
@@ -61,19 +61,24 @@ def scroll(self, origin_el: WebElement, destination_el: WebElement, duration: Op
         actions.perform()
         return cast('WebDriver', self)
 
-    def drag_and_drop(self, origin_el: WebElement, destination_el: WebElement) -> 'WebDriver':
+    def drag_and_drop(
+        self, origin_el: WebElement, destination_el: WebElement, pause: Union[float, None] = None
+    ) -> 'WebDriver':
         """Drag the origin element to the destination element
 
         Args:
             origin_el: the element to drag
             destination_el: the element to drag to
+            pause: how long the action pauses before moving after the tap and hold, in float seconds.
 
         Returns:
             Union['WebDriver', 'ActionHelpers']: Self instance
         """
         actions = ActionChains(self)
         # 'mouse' pointer action
         actions.w3c_actions.pointer_action.click_and_hold(origin_el)
+        if pause:
+            actions.w3c_actions.pointer_action.pause(pause)
```

Comment:
I assume we need to convert the value to integer milliseconds according to w3c spec

Reply:
selenium client takes care it
https://github.com/SeleniumHQ/selenium/blob/93c780439f2886607474a6cebaf4e678f9730047/py/selenium/webdriver/common/actions/pointer_actions.py#L198-L200
https://github.com/SeleniumHQ/selenium/blob/93c780439f2886607474a6cebaf4e678f9730047/py/selenium/webdriver/common/actions/pointer_input.py#L63-L64